<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="editCel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Celsius</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinFahr">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-999</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCtoF">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Celsius to Fahrenheit &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFtoC">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt; Fahrenheit to Celsius</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConvert">
    <property name="title">
     <string>&amp;Convert</string>
    </property>
    <addaction name="actionC_to_F"/>
    <addaction name="actionF_to_C"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConvert"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionC_to_F">
   <property name="text">
    <string>&amp;C to F</string>
   </property>
  </action>
  <action name="actionF_to_C">
   <property name="text">
    <string>&amp;F to C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
